announcing-pydoc-io.rst

```diff
@@ -0,0 +1,93 @@
+.. post:: Nov 4, 2016
+   :tags: pydoc, documentation, api, reference
+
+Announcing pydoc.io beta
+========================
+
+Today we're excited to announce our latest project: https://pydoc.io.
+This work was made possible by the :doc:`MOSS Grant </rtd-awarded-mozilla-open-source-support-grant>`_ from Mozilla.
+Thanks to Mozilla for giving us the money to set aside time to build this wonderful community resource.
+
+About pydoc.io
+--------------
+
+Running Read the Docs,
+we've always been proud of the documentation tooling that the Python community has.
+We prioritize prose over API documentation listings,
+and generally have a high standard of documentation in our projects.
+
+However,
+there are a specific set of use cases that API reference documentation support,
+and the Python community doesn't support them well.
+Tools like http://godoc.org and http://rubydoc.info provide these for others languages,
+and we now introduce http://pydoc.io as the Python communities version of that.
```

Comment:
This paragraph is difficult to parse. I think this is because it repeatedly refers to `use cases` without explaining this at all. You dive in a little below, perhaps start up here.

Also: `communities` -> `community's`